### ISSUE TITLE
Update React boilerplate to v15

### ISSUE THREAD

**Issue by bebraw** [christianalfoni/webpack-bin#131]
Looks like the boilerplate is still running on 0.14. Other boilerplates might need tweaks as well.

Feature idea: Make it possible to point to a GitHub based boilerplate (the user passes an url and WebpackBin picks it up then).

**Comment by christianalfoni** [comment]
Yeah, that would be great. When Users are implemented it will be a lot easier to handle boilerplates. Doing it manually now. Will change this as soon as I get a chance :)
